src/main/global.gd

```diff
@@ -11,9 +11,6 @@ const NUM_SCANCODES: Dictionary = {
 # Target number of creature greetings (hello, goodbye) per minute
 const GREETINGS_PER_MINUTE := 3.0
 
-# The scenario currently being launched or played
-var scenario_settings := ScenarioSettings.new()
-
 # The scenario which was initially launched. Some tutorial scenarios transition
 # into other scenarios, so this keeps track of the original.
 var launched_scenario_name
```

Comment:
This should probably stay paired with `scenario_settings` since they're so closely related

Reply:
Moved `overworld_puzzle` and `launched_scenario_name` into Scenario.